operations/productcategories.md

```diff
@@ -0,0 +1,123 @@
+| `ServiceIds` | array of string | required, max 1000 items | Unique identifiers of [Services](services.md#service) to which the resource categories belong. |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the product categories were updated. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned. |
+
+### Response
+
+```javascript
+{
+    "ProductCategories":[
+        {
+            "Id": "5c0804f9-d03a-4b13-a57d-b00300781a41",            
+            "ServiceId": "9b3a6c54-63aa-4383-b50e-b0030078184b",            
+            "Names": {
+                "en-US": "Alcohol 1"
+            },
+            "ShortNames": {
+                "en-US": "Alcohol 1"
+            },
+            "Descriptions": {},
+            "ParentProductCategory": [
+                "Id": "0b82f6d8-e5eb-4711-9069-40db2b8d968f",
+                "Names": {
+                    "en-US": "Alcoholic beverages"
+                },
+            ],
+            "CreatedUtc": "2023-06-09T13:32:32Z",
+            "UpdatedUtc": "2023-06-09T13:32:32Z",
+            "Ordering": 1,
+        },
+        {
+            "Id": "f0709b02-f1a4-46b2-9c1e-744b5ecd6980",            
+            "ServiceId": "9b3a6c54-63aa-4383-b50e-b0030078184b",            
+            "Names": {
+                "en-US": "Lemonade 1"
+            },
+            "ShortNames": {
+                "en-US": "Lemonade 1"
+            },
+            "Descriptions": {},
+            "ParentProductCategory": [
+                "Id": "aa4f3930-3f9d-4df7-9c6b-5aba5b2f6f7e",
+                "Names": {
+                    "en-US": "Lemonades"
+                },
+            ],
+            "CreatedUtc": "2023-06-09T13:32:32Z",
+            "UpdatedUtc": "2023-06-09T13:32:32Z",
+            "Ordering": 1,
+        }
+    ],
+    "Cursor": "f0709b02-f1a4-46b2-9c1e-744b5ecd6980"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ProductCategories` | array of [Product category](#product-category) | optional | Product categories. |
+| `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
+
+#### Product category
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the category. |
+| `EnterpriseId` | string | required | Unique identifier of the [Enterprise](enterprises.md#enterprise). |
+| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) of the resource category. |
+| `Names` | [Localized text](#localized-text) | required | All translations of the name. |
```

Comment:
Is [Localized text](#localized-text) defined in this page? I can't see it anywhere.

Reply:
Nope, my bad... and since it's on more places, I've moved it to `_objects.md`. I think that is a place where it should be now.